docs/src/increase-observability/logs/forward-logs.md

```diff
@@ -0,0 +1,123 @@
+
+View your logs in the **Apps->Search & Reporting** dashboard.
+Filter by the index name to find the relevant events.
+
+<--->
+---
+title=Sumo Logic
+highlight=false
+file=none
+---
+
+1. In Sumo Logic, [configure an HTTP source](https://help-opensource.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/#configure-an-httplogs-and-metrics-source).
+   Make sure to copy the Source Category and collector URL.
+2. Create the integration with the following command:
+
+   <div class="highlight"><pre tabindex="0" class="chroma"><code class="language-bash" data-lang="bash"><span class="line"><span class="cl">platform project:curl /integrations -X POST -d <span class="s1">'{ "type":"sumologic", "url":"{{< variable "COLLECTOR_URL" >}}", "category":"{{< variable "SOURCE_CATEGORY" >}}" }'</span></span></span></code></pre></div>
+
+View your logs in the **Log Search** tab.
+
+{{< /codetabs >}}
+
+Logs start getting forwarded automatically after the integration has been resolved,
+which can take up to an hour.
+
+## Forward to a syslog endpoint
+
+Syslog is a standard for message logging.
+Many third-party services offer endpoints for ingesting syslog events.
+You can forward your Platform.sh logs to any of those endpoints.
+
+To enable this forwarding, run a command like the following:
+
+```bash
+platform project:curl /integrations -X POST --data '{ "type":"syslog", "host":"{{< variable "INGESTION_HOST" >}}", "port":"{{< variable "INGESTION_HOST_PORT" >}}" }'
+```
+
+`type`, `host`, and `port` are the only properties required for all endpoints.
+The following table shows the other available properties:
+
+| Property         | Type      | Default    | Description |
+| -------- --------| --------- | ---------- | ----------- |
+| `protocol`       | `string`  | `tls`      | The network protocol to use in the connection. Can be one of `tls`, `tcp`, or `udp`. Defaults to `tls`. |
+| `facility`       | `string`  | `1` (user) | A [syslog facility code](https://en.wikipedia.org/wiki/Syslog#Facility) to attach with each log to identify the source. Can be a number from 0 to 23. |
+| `message_format` | `string`  | `rfc5424`  | The standard to use for the message format. Can be `rfc5424` or `rfc3164`. |
+| `auth_token`     | `string`  |            | The token to authenticate with the given service. |
+| `auth_mode`      | `string`  |            | The mode for authentication with the given service. Can be `prefix` or `structured_data`. |
+| `tls_verify`     | `boolean` | `true`     | Whether to verify Transport Layer Security (TLS) certification when using the TLS protocol. |
+
+Place any property you want inside the `data` object of the cURL request.
+This should let you connect to any service that has syslog endpoints.
+
```

Comment:
😍 I think having the standard information in the table + the bulleted list with links to other docs works for now (cf conversation on Slack). We can change strategy later if we get feedback that suggests it's not helpful enough.

I understand that the bulleted list is pulled from another file, I just can't find exactly where I'm supposed to tell you that Datadog is the only link that is followed by a colon, so I'd suggest changing it to a hyphen to fit the general style of the list.

Reply:
You probably can't find the place because I deleted the list :grinning:

Reply:
It happens when I push changes after requesting a review :speak_no_evil: